src/manager/YaraManager.js

```diff
@@ -47,7 +47,7 @@ export default class YaraManager extends AsyncWorker {
    *      matchedRule: only valid if (isMalicious == true)
    *      isMalicious: boolean, null if unknown
    */
-  async setupResourceScan(jsArtifacts, ticket) {
+  async setupResourceScan(Resources, ticket) {
```

Comment:
Try to reserve proper case variable names for classes. `Resources` -> `resources`.